docs/roadmap.rst

```diff
@@ -0,0 +1,111 @@
+##########################################
+Rubin Science Platform Status and Roadmap
+##########################################
+
+What can you do with the Rubin Science Platform?
+================================================
+
+Below you will find a detailed (but not exhaustive) list of current and future RSP capabilities.
+
+Please note the difference between Data Previews (DP0, DP1, DP2), which are early access programs before the start of the survey, and Data Releases (DR1, DR2, etc.), which are the official data releases after the survey starts.
+For the planned dates for those events please consult project documentation and announcements.
+
+But before we get there, bear in mind there are three constraints to what you can do on the RSP:
```

Comment:
> But before we get there,

Note our documentation style is not to use "we" as you would in academic writing. Our user documentation style aims to be casual but efficient and this is maybe verging on too chatty.

Reply:
I will award myself some kind of medal for being too academic and too chatty in the same sentence.